ansible/inventory/env/group_vars/all.yml

```diff
@@ -483,74 +483,6 @@ sunbird_otp_expiration: 1800
 sunbird_otp_length: 6
 sunbird_help_link_visibility: FALSE
 
-
-
-#Container Limits
-proxy_replicas: 1
-proxy_reservation_memory: 64M
-proxy_limit_memory: 128M
-kong_replicas: 1
-kong_reservation_memory: 64M
-kong_limit_memory: 256M
-echo_service_replicas: 1
-echo_service_reservation_memory: 8M
-echo_service_limit_memory: 16M
-adminutil_replicas: 1
-adminutil_reservation_memory: 300M
-adminutil_limit_memory: 300M
-learner_limit_cpu: 1
-learner_replicas: 1
-learner_reservation_memory: 1000M
-learner_limit_memory: 1000M
-logger_logstash_heap_size: 512M
-logger_logstash_replicas: 1
-logger_logstash_reservation_memory: 512M
-logger_logstash_limit_memory: 750M
-logger_kibana_reservation_memory: 750M
-logger_kibana_limit_memory: 750M
-logger_logspout_reservation_memory: 200M
-logger_logspout_limit_memory: 200M
-logger_oauth_reservation_memory: 32M
-logger_oauth_limit_memory: 32M
-prometheus_reservation_memory: 1G
-prometheus_limit_memory: 1G
-alertmanager_reservation_memory: 100M
-alertmanager_limit_memory: 100M
-node_exporter_reservation_memory: 16M
-node_exporter_limit_memory: 32M
-cadvisor_reservation_memory: 100M
-cadvisor_limit_memory: 100M
-elasticsearch_exporter_reservation_memory: 8M
-elasticsearch_exporter_limit_memory: 24M
-postgres_exporter_reservation_memory: 16M
-postgres_exporter_limit_memory: 32M
-statsd_exporter_reservation_memory: 8M
-statsd_exporter_limit_memory: 16M
-blackbox_exporter_reservation_memory: 16M
-blackbox_exporter_limit_memory: 32M
-jsonpath_exporter_reservation_memory: 32M
-jsonpath_exporter_limit_memory: 64M
```

Comment:
@keshavprasadms if we remove all these what will be the default value for adopters ?

Reply:
Moved to respective roles and monitoring branch